layouts/partner/aws.html

```diff
@@ -103,7 +116,7 @@ <h3 class="mt-12 mb-8 text-center">
                      during the re:Invent takeover. Once we publish another blog we should remove
                      this exclusion.
                 -->
-                {{ range first 3 ( where $blog ".Params.title" "ne" "Deploying Minecraft on Azure" ) }}
+                {{ range first 3 ( where ( where $blog ".Params.title" "ne" "Deploying Minecraft on Azure" ) ".Params.title" "ne" "Get Up and Running with Azure Synapse and Pulumi") }}
```

Comment:
This doesn't seem maintainable. Is there a better way to filter these? Maybe by an additional frontmatter param?

Reply:
This section is going to be removed post re:Invent, so it should be fine to leave this as-is for now.